openfisca_core/formulas.py

```diff
@@ -694,7 +695,7 @@ def new_filled_column(base_function = UnboundLocalError, calculate_output = Unbo
     assert isinstance(name, unicode)
 
     if calculate_output is UnboundLocalError:
-        calculate_output = None if reference_column is None else reference_column.formula_class.calculate_output
+        calculate_output = None if reference_column is None else reference_column.formula_class.calculate_output.im_func
```

Comment:
It is the implementation function and not the method related to the reference_column_formula_calss. Ask @eraviart for more technical details ! ;-)